File: sort/selection_sort.py
def selection_sort(a):
    for i in range(len(a)):
        min_index, min_val = i, a[i]
        for j in range(i, len(a)):
            if a[min_index] > a[j]:
                min_index = j
        a[i], a[min_index] = a[min_index], a[i]

File: sort/selection_sort.c
void selection_sort(int arr[], int n){
    int i, j, tmp;
    for (i=0; i<n-1; i++) {
        int minIndex = i;
        for (j=i + 1; j<n; j++) {
            if (arr[minIndex] > arr[j]) {
                minIndex = j;
            }
        }
        tmp = arr[i];
        arr[i] = arr[minIndex];
        arr[minIndex] = tmp;
    }
}
